summit.html

```diff
@@ -109,10 +68,418 @@ <h1>Agenda</h1>
+                      suitable.
+                    </p>
+                    <p>
+                      <b>Speaker:</b> Eiffel TC &amp; Erik Sternersson
+                    </p>
+                  </div>
+                </details>
+              </td>
+              <td><span class="tag bd-success">Beginner</span></td>
+              <td>9:00</td>
+            </tr>
+
+            <tr>
+              <td>
+                <details>
+                  <summary>Software Center project update</summary>
+                  <div>
+                    <p>
+                      We will hear about the latest status on the software
+                      center project about visualization and also about
+                      learning chains of events.
+                    </p>
+                    <p>
+                      <b>Speaker:</b> Kristian Sandahl
+                    </p>
+                  </div>
+                </details>
+              </td>
+              <td><span class="tag bd-success">Intermediate</span></td>
+              <td>9:15</td>
+            </tr>
+
+            <tr>
+              <td>
+                <details>
+                  <summary>CDF, CDEvents and Secure SW Supply Chains</summary>
+                  <div>
+                    <p>
+                      <ul>
+                        <li>What is the Continuous Delivery Foundation?</li>
+                        <li>What is CDEvents, what is its current status and how
+                          does it relate to Eiffel?</li>
+                        <li>How could events improve security in the Software
+                          Supply Chains?</li>
+                      </ul>
+                    </p>
+                    <p>
+                      <b>Speakers:</b> Fatih Degirmenci &amp; Emil Bäckmark
+                    </p>
+                  </div>
+                </details>
+              </td>
+              <td><span class="tag bd-success">Intermediate</span></td>
+              <td>9:30</td>
+            </tr>
+
+            <tr><td>Break</td><td></td><td>10:30</td></tr>
+
+            <tr>
+              <td>
+                <details>
+                  <summary>Submit events securely</summary>
+                  <div>
+                    <p>
+                      A proposal for securely submitting events
+                    </p>
+                    <p>
+                      <b>Speaker:</b> Magnus Bäck
+                    </p>
+                  </div>
+                </details>
+              </td>
+              <td><span class="tag bd-success">Advanced</span></td>
+              <td>10:45</td>
+            </tr>
+
+            <tr>
+              <td>
+                <details>
+                  <summary>CLM for HW &amp; SW Combined</summary>
+                  <div>
+                    <p>
+                      How to set the confidence level on a combination of
+                      hardware and software?
```

Comment:
```suggestion
                      Eiffel describes software, but some software can be used
                      on multiple types of hardware. When we express
                      confidence in a piece of software, how can we also
                      capture and describe the combination of software and
                      hardware that we're feeling confident of? This is a
                      follow-up of the
                      <a href="https://hackmd.io/dpQQOkEeQoC5yV7IpFUYOg">
                        2023-04-13 community meeting
                      </a>.
```

Reply:
Will update to something along those lines